api/src/main/java/org/pmiops/workbench/db/jdbc/ReportingQueryServiceImpl.java

```diff
@@ -228,6 +228,8 @@ public List<ReportingUser> getUsers(long limit, long offset) {
                 + "  uame.era_commons_completion_time,\n"
                 + "  uaml.ras_login_gov_bypass_time,\n"
                 + "  uaml.ras_login_gov_completion_time,\n"
+                + "  uami.identity_bypass_time,\n"
```

Comment:
Is there any problem with sending the reporting team more data than they are expecting?